test/rates_test.rb

```diff
@@ -57,6 +57,9 @@ def test_exchange_rate
 
     assert_equal 5.944602, fx.exchange_rate(:from => "AUD", :to => "HRK")
     assert_equal 0.168220, fx.exchange_rate(:from => "HRK", :to => "AUD")
+
+    assert_equal 0.00023, fx.exchange_rate(:from => 'SLL', to: 'USD')
+    assert_equal 4350, fx.exchange_rate(:from => 'USD', to: 'SLL')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.